backend/clubs/models.py

```diff
@@ -1592,6 +1592,18 @@ def validate_template(cls, template):
         return all(t in cls.VALID_TEMPLATE_TOKENS for t in tokens)
 
 
+class ApplicationExtension(models.Model):
```

Comment:
Consider a [unique_together](https://docs.djangoproject.com/en/4.2/ref/models/options/#unique-together) constraint on `user` and `application`